src/components/pages/pages.stories.mdx

```diff
@@ -3,7 +3,7 @@ import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
 import DefaultPages from './pages.component';
 import Page from './page/page.component';
 import DialogFullScreen from '../dialog-full-screen';
-import Heading from '../heading/heading';
+import Heading from '../heading';
 import Button from '../button';
 
```

Comment:
The other visual defects have been rectified but this visual defect has appeared with `Pages` now.

<img width="1473" alt="Screenshot 2021-01-28 at 11 19 28" src="https://user-images.githubusercontent.com/56251247/106131515-b2d22700-615a-11eb-9243-2b9e201f285e.png">